.prettierrc.cjs

```diff
@@ -0,0 +1,6 @@
+/** @type {import('prettier').Config} */
+module.exports = {
+  singleQuote: true,
+  printWidth: 100,
```

Comment:
Prettier default has 80 chars, I usually bump it up to 100.

We can update it to mirror the default, up to you